src/Orchard.Web/Core/Contents/Views/Admin/Edit.cshtml

```diff
@@ -2,7 +2,7 @@
     var typeDisplayName = Model.ContentItem.TypeDefinition.DisplayName;
     var pageTitle = T("Edit Content");
     if (!string.IsNullOrWhiteSpace(typeDisplayName)) {
-        pageTitle = T("Edit {0}", typeDisplayName);
+        pageTitle = T("Edit {0}", Html.Raw(typeDisplayName));
```

Comment:
Are you sure it's necessary? The display name should not have been encoded by anything.